earth_enterprise/integration_test/specs/minification.spec

```diff
@@ -0,0 +1,50 @@
+
+text underlined with = chars denotes the spec name
+text underlined with - chars denotes the scenario name
+text that starts with * chars denote steps
+
+' Guideline for format:
+PRE-TEST steps go here.   These steps will be run before EACH and EVERY test (scenario)
+' * Example setup step to be performed.
+
+
+Example for scenario including cleanups (aka POST-STEP):
+' * Example clean up steps to be performed.
+' -----------------------------------------
+' Tags: Comma separated list of Tag Names
+' * Example steps
+' -----------------------------------------
+' POST-STEPs for this SCENARIO go here.
+
+'##########################################
+'# DO NOT DELETE ANYTHING ABOVE THIS LINE #
+'##########################################
+
+
+Open GEE New Imagery Resource Command Test
```

Comment:
Should change the title to refer to minification

Reply:
I was actually referring to the line above this, which refers to the "New Imagery Resource Command", which was copied from another test. It should mention minification instead.